### PR TITLE
Skeleton basics

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

**Comment by justinvdm** [comment]
Going to quickly improve the readme a bit.

**Comment by justinvdm** [comment]
readme improved, actually ready for review now.

File: test/app.test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                        reply: [
+                            'Hi there! What do you want to do?',
+                            '1. Show this menu again',
+                            '2. Exit'
+                        ].join('\n')
+                    })
+                    .run();
+            });
+
+            describe("when the user asks to see the menu again", function() {
```
Comment (hodgestar) [inline test/app.test.js]: I think this should be one indentation / block-level higher up? Or maybe I'm misunderstanding the structure?

**Comment by hodgestar** [comment]
Other than the question about the test indentation, looks good.

File: test/app.test.js (hunk 1 above)
Comment (justinvdm) [inline test/app.test.js]: This is the intended structure, but i'd be happy to try put these up a level.

**Comment by justinvdm** [comment]
Ready for another review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**